.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,7 @@
 <!-- Learn the overall process and best practices for pull requests at https://github.com/WordPress/gutenberg/blob/HEAD/docs/contributors/repository-management.md#pull-requests. -->
 
+<!-- Gutenberg's license is in the process of updating to be dual-licensed under the GPL and MPL. As part of that transition, all new contributions must be made under that dual-license: https://github.com/WordPress/gutenberg/blob/trunk/LICENSE.md -->
```

Comment:
IANAL so, not sure what the `must` in the `must be made under` actually means for the contributor about to open a PR. I mean, while reading as a contributor, it I'm not sure if I need to do anything special to comply. Would replacing the `must be` with `are` can perhaps make it more clear that the text is just stating a fact and there's nothing the reader needs or have to do? Like: `...all new contributions must be made under that dual-license:`.

Also, in the same spirit, do we need to make it clear that "Submitting a PR automatically puts the code under the dual-license." or similar?

Reply:
You're right, this was a bit unclear. We don't need to solicit any action from the PR author here, I just want to try and ensure that they are aware of the license change and the fact that their contribution would be dual-licensed. I simplified the language a little bit to make that clearer. Let me know what you think.

Reply:
Ah, changes are working for me, thanks! 👍